infra/app/database/main.tf

```diff
@@ -87,9 +59,9 @@ module "database" {
   migrator_username = local.database_config.migrator_username
   schema_name       = local.database_config.schema_name
 
-  vpc_id                         = data.aws_vpc.network.id
-  database_subnet_group_name     = local.network_config.database_subnet_group_name
-  private_subnet_ids             = data.aws_subnets.database.ids
-  aws_services_security_group_id = data.aws_security_groups.aws_services.ids[0]
+  vpc_id                         = module.network.vpc_id
```

Comment:
Would it make sense to call the network module from within the database module instead? 

Or are we intentionally avoiding nesting any modules unless strictly required? (eg. project config inside of app config)

Reply:
The primary issue isn't with nested modules, it's the fact that calling module.network requires the project name. The reusable modules in infra/modules/ should not rely on anything project related like project-config or app-config since they are meant to be reusable.